bentoml/saved_bundle/bentoml-init.sh

```diff
@@ -1,57 +1,73 @@
 #!/usr/bin/env bash
 # Bash Script that installs the dependencies specified in the BentoService archive
+#
+# Usage:
+#   * `bentoml-init.sh` to run the full script
+#   * `bentoml-init.sh <step_name>` to run a specific step 
+#      available steps: [setup.py python_version environment.yml requirement.txt bundled_pip_dependencies]
+
 set -ex
 
 # cd to the saved bundle directory
 SAVED_BUNDLE_PATH=$(cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P)
 cd $SAVED_BUNDLE_PATH
 
 # Run the user defined setup.sh script if it is presented
-if [ -f ./setup.sh ]; then chmod +x ./setup.sh && bash -c ./setup.sh; fi
+if [ $# -eq 0 ] || [ $1 == "setup.py" ] ; then
```

Comment:
Overall looks great! Could you update the step names to a more descriptive name as we discussed? Otherwise, I think this PR is ready to go!

Reply:
done.